tdrs-frontend/Dockerfile.base

```diff
@@ -1,16 +1,9 @@
-
```

Comment:
still installs all dev dependencies (pipenv install --dev --system --deploy) in the builder and copies them into the runtime. That ships tooling (linters, formatters, etc.) in the final image and keeps the image large; it also defeats the stated image-size reduction goal.